shell/platform/embedder/embedder.cc

```diff
@@ -1230,9 +1291,44 @@ CreateEmbedderRenderTarget(
             break;
           }
         }
+
+        case kFlutterOpenGLTargetTypeSurface: {
+          auto on_make_current =
+              [callback = backing_store.open_gl.surface.make_current_callback,
+               context = backing_store.open_gl.surface.user_data]()
+              -> std::pair<bool, bool> {
+            bool invalidate_api_state = false;
+            bool ok = callback(context, &invalidate_api_state);
+            return {ok, invalidate_api_state};
+          };
+
+          auto on_clear_current =
+              [callback = backing_store.open_gl.surface.clear_current_callback,
+               context = backing_store.open_gl.surface.user_data]()
+              -> std::pair<bool, bool> {
+            bool invalidate_api_state = false;
+            bool ok = callback(context, &invalidate_api_state);
+            return {ok, invalidate_api_state};
+          };
+
+          if (enable_impeller) {
+            /// TODO: Implement
+            FML_LOG(ERROR) << "Unimplemented";
+            break;
```

Comment:
This and the `break` below seem unnecessary given that there's no code after either of these either within their respective conditional blocks or after them, before the top-level break for the `case`.

Reply:
I think (if I see it correctly) there is actually no catch-all break for this `case`, the next-closest `break` is for the `case` of the outer switch statement. I think that's why the breaks are as far in as possible in this whole function, because it's really hard to see the structure

Reply:
I was assuming the one on line 1331 (old) 1332 (new) was it, but you're right... that's out a couple more levels of indentation. Thanks for pushing back!

I suppose what this really indicates is this code is probably overdue for some refactoring and extracting out functions in some later patch.